frontend/public/components/sidebars/resource-sidebar-samples.tsx

```diff
@@ -228,6 +232,31 @@ const defaultSamples = ImmutableMap<GroupVersionKind, Sample[]>()
         targetResource: getTargetResource(ConsoleLinkModel),
       },
     ],
+  )
+  .setIn(
+    [referenceForModel(ConsoleModel)],
+    [
+      {
+        title: 'Add catalog categories',
+        description:
+          'Provides a list of default categories which are shown in the Developer Catalog.',
+        id: 'default',
+        snippet: true,
+        lazyYaml: () => {
+          const yaml = {
+            spec: {
+              customization: {
+                developerCatalog: {
+                  categories: defaultCatalogCategories,
+                },
+              },
+            },
```

Comment:
I think if the snippet should be `defaultCatalogCategories` instead of a full spec.
This would allow someone to insert just the categories themselves.
Perhaps the description should be used to help guide the user where the snippet belongs

Reply:
Changed this to 

```tsx
return YAML.dump(defaultCatalogCategories);
```

Because the editor keeps indentation fine.